src/Keep.lua

```diff
@@ -326,10 +357,7 @@ local function transformUpdate(keep: Keep, newestData: KeepStruct, release: bool
 			local currentGlobals = latestKeep.GlobalUpdates
 			local newGlobals = newestData.GlobalUpdates
 
-			local finalGlobals = {
-				ID = 0,
-				Updates = {},
-			} -- the final global updates to save
+			local finalGlobals = { ID = 0, Updates = {} } -- the final global updates to save
```

Comment:
These spanned out tables may always exist. I have stylua to auto format tables for easier to read